docs/user/rstudio/ide/guide/tools/terminal.qmd

```diff
@@ -2,93 +2,100 @@
 title: "Terminal"
 ---
 
-Adapted from: <https://support.rstudio.com/hc/en-us/articles/115010737148-Using-the-RStudio-Terminal-in-the-RStudio-IDE>
-
-!!!NOTE LEFT OUT APPENDIX AND ADVANCED SECTIONS
-
 The RStudio terminal provides access to the system shell from within the RStudio. It supports full-screen terminal applications such as vim, Emacs, and tmux as, well as regular command-line operations with line-editing and shell history.
 
-Potential uses of the shell including advanced source control operations, execution of long-running jobs, remote logins, system administration of RStudio Workbench or RStudio Server, and the aforementioned full-screen terminal programs.
+Potential uses of the shell including advanced source control operations, execution of long-running jobs, remote logins, system administration of Posit Workbench or RStudio Server, and the aforementioned full-screen terminal programs.
 
 This article does not attempt to cover fundamentals on when and how to use a system shell. Instead, it focuses on the features, design, options, and limitations of the RStudio terminal feature itself.
 
-Unlike most RStudio features, the nature of the terminal leads to several platform-specific differences. These primarily break down into two categories: Windows, and everything else (Mac, Linux, and Server). [Appendix A](https://support.rstudio.com/hc/en-us/articles/115010737148-Using-the-RStudio-Terminal-in-the-RStudio-IDE#appendixa) provides a summary.
+Unlike most RStudio features, the nature of the terminal leads to several platform-specific differences. These primarily break down into two categories: Windows, and everything else (Mac, Linux, and Server). Windows provides emulators for unix-based terminals such as Git Bash, Windows Subsystem for Linux, or PowerShell.
 
 ## Getting Started
 
 The **Terminal tab** is next to the **Console tab**. Click on the Terminal tab and a new terminal session will be created (if there isn't one already).
 
--   If the tab isn't visible, show it via **Shift+Alt+T** (**Tools-\>Terminal-\>Move Focus to Terminal**).
+-   If the tab isn't visible, display it via **Tools** \> **Terminal** \> **Move Focus to Terminal** (or use the keyboard shortcut <kbd>Shift+Alt+T</kbd>).
```

Comment:
the linked [keyboard shortcut page](https://colorado.rstudio.com/rsc/rstudio-user-guide-2022-09-29/ide/reference/shortcuts.html#terminal) has this as shift + alt + m

Reply:
Confirmed that Alt+Shift+M is the default - changed, thank you!